crates/re_viewer/src/misc/transform_cache.rs

```diff
@@ -61,6 +60,13 @@ impl std::fmt::Display for UnreachableTransform {
     }
 }
 
+fn transform_affine3(a: glam::Affine3A, b: glam::Affine3A) -> glam::Affine3A {
+    glam::Affine3A {
+        matrix3: a.matrix3.mul_mat3(&b.matrix3),
+        translation: a.matrix3.mul_vec3a(b.translation) + a.translation,
+    }
+}
```

Comment:
Just use https://docs.rs/glam/latest/src/glam/f32/affine3a.rs.html#490-500

Reply:
huh. didn't find this before, I swear I got an error on multiply....